### ISSUE TITLE
RBAC roles are not applicable by the default admin cluster role

### ISSUE THREAD

**Issue by pawelprazak** [Azure/brigade#327]
Expected:

- be able to install Helm charts with RBAC enabled using the [default `admin` ClusterRole](https://kubernetes.io/docs/admin/authorization/rbac/#user-facing-roles), e.g. binding:

```
kind: RoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: tiller-binding
  namespace: default
subjects:
- kind: ServiceAccount
  name: tiller
  namespace: default
roleRef:
  kind: ClusterRole
  name: admin
  apiGroup: rbac.authorization.k8s.io
```

Current:
- RBAC roles in current state are not usable with the default `admin` ClusterRole.

There is only one small problem with `pods/log`, part of error message:
```
Error: release brigade failed: roles.rbac.authorization.k8s.io "brigade-brigade-wrk" is forbidden: attempt to grant extra privileges:
[PolicyRule {
Resources: ["pods/log"], APIGroups:[""], Verbs:["create"]
} 
PolicyRule{
Resources:["pods/log"], APIGroups:[""], Verbs:["update"]
} 
PolicyRule{
Resources:["pods/log"], APIGroups:[""], Verbs:["patch"]
} 
PolicyRule{
Resources:["pods/log"], APIGroups:[""], Verbs:["delete"]
}] 
```

IMO it makes perfect sense NOT to use `create`, `update`, `patch` or `delete` on logs, only `read`.

I suspect it's a simple matter of adjusting RBAC role templates, since those verbs are probably not used anyway, but I couldn't be sure without reading all the code.

**Comment by technosophos** [comment]
@pawellipiec is this one fixed now with the merge of #331?

**Comment by pawelprazak** [comment]
Yes :)
